pkg/platform/api/buildplanner/request/stagecommit.go

```diff
@@ -4,10 +4,9 @@ import (
 	"time"
 
 	"github.com/ActiveState/cli/internal/rtutils/ptr"
-	"github.com/ActiveState/cli/pkg/platform/runtime/buildexpression"
 )
 
-func StageCommit(owner, project, parentCommit, description string, atTime *time.Time, expression *buildexpression.BuildExpression) *buildPlanByStageCommit {
+func StageCommit(owner, project, parentCommit, description string, atTime *time.Time, expression []byte) *buildPlanByStageCommit {
```

Comment:
Same question: what do you think about making `expression` a string type instead of `[]byte`?